From foo@baz Tue Apr  9 12:12:43 2002
To: Greg KH <greg@kroah.com>
Date: 07 Feb 2005 14:37:42 -08:00
From: eike-hotplug@sf-tec.de
Subject: [PATCH] PCI Hotplug: Remove unneeded instructions from ibmphp_pci.c

this patch removes some unneeded code from ibmphp_pci.c. First I thought it
is a bug and the second line should have been "sec_no = (int) sec_number".
But than I found exactly the same read only 9 lines higher and after it the
line I expected the second one to be. Between the 2 pci_bus_read_config_byte's
are only some checks so I don't expect them to return different results. And
sec_no is and int so removing this wont change anything at all.


Signed-off-by: Rolf Eike Beer <eike-hotplug@sf-tec.de>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


diff -Nru a/drivers/pci/hotplug/ibmphp_pci.c b/drivers/pci/hotplug/ibmphp_pci.c
--- a/drivers/pci/hotplug/ibmphp_pci.c	2005-03-02 23:05:24 -08:00
+++ b/drivers/pci/hotplug/ibmphp_pci.c	2005-03-02 23:05:24 -08:00
@@ -1384,9 +1384,6 @@
 		return -EINVAL;
 	}
 
-	pci_bus_read_config_byte (ibmphp_pci_bus, devfn, PCI_SECONDARY_BUS, &sec_number);
-	sec_no = (int) sec_no;
-
 	pci_bus_read_config_byte (ibmphp_pci_bus, devfn, PCI_SUBORDINATE_BUS, &sub_number);
 	sub_no = (int) sub_number;
 	debug ("sub_no is %d, sec_no is %d\n", sub_no, sec_no);
